e2e/client/playwright/templates.spec.ts

```diff
@@ -0,0 +1,117 @@
+import {restoreDatabaseSnapshot, s} from './utils';
+
+test('creating new template', async ({page}) => {
+    await restoreDatabaseSnapshot();
+    await page.goto('/#/settings/templates');
+
+    await page.locator(s('template-header')).getByRole('button', {name: 'Add new'}).click();
+
+    await page.locator(s('template-edit-view')).getByPlaceholder('template name').fill('Template 1');
+    await page.locator(s('template-edit-view')).getByLabel('Content Profile').selectOption({label: 'Story'});
+    await page.locator(s('template-edit-view')).getByRole('button', {name: 'Save'}).click();
+
+    await expect(page.locator(s('template-content', 'content-template=template 1'))).toBeVisible();
+});
+
+test('editing template name', async ({page}) => {
+    await restoreDatabaseSnapshot();
+    await page.goto('/#/settings/templates');
+
+    await page.locator(s('template-content', 'content-template=story', 'template-actions')).click();
+    await page.locator(s('template-actions--options')).getByRole('button', {name: 'Edit'}).click();
+    await page.locator(s('template-edit-view')).getByPlaceholder('template name').fill('story 1.1');
+    await page.locator(s('template-edit-view')).getByRole('button', {name: 'Save'}).click();
+
+    await expect(page.locator(s('template-content', 'content-template=story 1.1'))).toBeVisible();
+});
+
+test('removing template', async ({page}) => {
+    await restoreDatabaseSnapshot();
+    await page.goto('/#/settings/templates');
+
+    await page.locator(s('template-content', 'content-template=story 2', 'template-actions')).click();
+    await page.locator(s('template-actions--options')).getByRole('button', {name: 'Remove'}).click();
+    await page.locator(s('modal-confirm')).getByRole('button', {name: 'Ok'}).click();
+    await expect(page.locator(s('template-content', 'content-template=story 2'))).not.toBeVisible();
+});
+
+test('assigning template to a desk', async ({page}) => {
+    const monitoring = new Monitoring(page);
+
+    await restoreDatabaseSnapshot();
+    await page.goto('/#/workspace/monitoring');
+    await monitoring.selectDeskOrWorkspace('Finances');
+
+    await page.locator(s('content-create')).click();
+    await page.locator(s('content-create-dropdown')).getByRole('button', {name: 'More Templates...'}).click();
+    await page.locator(s('content-create-dropdown', 'search')).fill('Story 2');
+    await expect(page.locator(s('content-create-dropdown')).getByRole('button', {name: 'Story 2'})).not.toBeVisible();
+
+    // assign template to the desk
+    await page.goto('/#/settings/templates');
+    await page.locator(s('template-content', 'content-template=story 2', 'template-actions')).click();
+    await page.locator(s('template-actions--options')).getByRole('button', {name: 'Edit'}).click();
+    await page.locator(s('template-edit-view', 'desks', 'desk--Finances')).click();
+    await page.locator(s('template-edit-view')).getByRole('button', {name: 'Save'}).click();
```

Comment:
could we somehow wait to ensure that saving finished before navigating away? maybe we could wait until modal closes?